foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-PostgreSQLPolicySetDefinition.json

```diff
@@ -17,47 +17,343 @@
+                                                        }
+                                                    },
+                                                    {
+                                                        "condition": "[[startsWith(parameters('resourceType'),'Microsoft.DBforPostgreSQL/servers')]]",
+                                                        "type": "Microsoft.DBforPostgreSQL/servers/providers/diagnosticSettings",
+                                                        "apiVersion": "2021-05-01-preview",
+                                                        "name": "[[concat(parameters('resourceName'), '/', 'Microsoft.Insights/', parameters('profileName'))]]",
+                                                        "location": "[[parameters('location')]]",
+                                                        "dependsOn": [],
+                                                        "properties": {
+                                                            "workspaceId": "[[parameters('logAnalytics')]]",
+                                                            "metrics": [
+                                                                {
+                                                                    "category": "AllMetrics",
+                                                                    "enabled": "[[parameters('metricsEnabled')]]",
+                                                                    "retentionPolicy": { "days": 0, "enabled": false },
+                                                                    "timeGrain": null
+                                                                }
+                                                            ],
+                                                            "logs": [
+                                                                {
+                                                                    "category": "PostgreSQLLogs",
+                                                                    "enabled": "[[parameters('logsEnabled')]]"
+                                                                },
+                                                                {
+                                                                    "category": "QueryStoreRuntimeStatistics",
+                                                                    "enabled": "[[parameters('logsEnabled')]]"
+                                                                },
+                                                                {
+                                                                    "category": "QueryStoreWaitStatistics",
+                                                                    "enabled": "[[parameters('logsEnabled')]]"
+                                                                }
+                                                            ]
```

Comment:
For logs you can also use categoryGroups these days. Can you please change that?